### DISCUSSION TITLE
About the amount of data used

### DISCUSSION THREAD

**Discussion by kin0303** [voicepaw/so-vits-svc-fork#235] [Q&A] [answered]
What is the minimum amount of data that can be used for training? And what is the minimum epoch?

**Comment by Lordmau5** [comment, marked as answer]
During my testing with a few characters I've found that it can vary.

Some can work well with only ~30s of voice clips and sound very good after only around 5000 iterations.
Others sound terrible with this little data after even 20k iterations. They need more data.

If you have *really clean voice clips* I'd say that between 2-5mins of data and a few thousand iterations can give good results.

Protip: Start training with `svc train -t` so it opens Tensorboard in your browser - you can head into the Audio tab at the top, then listen to the checkpoints that were made for the model (by default, every 800 steps / iterations)
